graphistry/feature_utils.py

```diff
@@ -2361,8 +2365,8 @@ def featurize(
                 #confidence=confidence,  # deprecated
                 min_words=min_words,
                 model_name=model_name,
-                #similarity=similarity,  # deprecated
-                #categories=categories,  # deprecated
+                similarity=similarity,  # deprecated
+                categories=categories,  # deprecated
```

Comment:
@silkspace if deprecated why in your fix?

Reply:
It is not Deprecated, it was my error -- I switched branches on VS code but it doesn't naturally switch branches on terminal where i do the checkin. I was playing with the idea of deprecating (and did a hot fix on Friday night so tests would pass).